src/state/orderPlacement/hooks.ts

```diff
@@ -216,32 +216,47 @@ export const useGetOrderPlacementError = (
   )
   const [balanceIn, amountIn] = [biddingTokenBalance, parsedBiddingAmount]
 
-  const amountMustBeBigger =
-    amountIn &&
-    price &&
-    price !== '0' &&
-    price !== 'Infinity' &&
-    derivedAuctionInfo?.minBiddingAmountPerOrder &&
-    derivedAuctionInfo?.biddingToken &&
-    sellAmount &&
-    (!sellAmountScaled ||
-      (sellAmountScaled &&
-        BigNumber.from(derivedAuctionInfo?.minBiddingAmountPerOrder).gte(sellAmountScaled)) ||
-      parseFloat(sellAmount) == 0) &&
-    `Amount must be bigger than
+  const amountMustBeBigger = () => {
+    if (!sellAmountScaled) return
+
+    return (
+      amountIn &&
+      price &&
+      price !== '0' &&
+      price !== '.' &&
+      price !== 'Infinity' &&
+      derivedAuctionInfo?.minBiddingAmountPerOrder &&
+      derivedAuctionInfo?.biddingToken &&
+      sellAmount &&
+      (!sellAmountScaled ||
+        (sellAmountScaled &&
+          BigNumber.from(derivedAuctionInfo?.minBiddingAmountPerOrder).gte(sellAmountScaled)) ||
+        parseFloat(sellAmount) == 0) &&
+      `Amount must be bigger than
       ${new Fraction(
         derivedAuctionInfo?.minBiddingAmountPerOrder,
         BigNumber.from(10).pow(derivedAuctionInfo?.biddingToken.decimals).toString(),
       ).toSignificant(2)}`
+    )
+  }
 
+  const invalidAmount = sellAmount && !amountIn && `Invalid Amount`
   const insufficientBalance =
     balanceIn &&
     amountIn &&
     balanceIn.lessThan(amountIn) &&
     `Insufficient ${getTokenDisplay(amountIn.token, chainId)}` + ' balance.'
 
   const priceEqualsZero =
-    amountIn && price && (price === '0' || price === 'Infinity') ? `Price must not be 0` : undefined
+    amountIn && price && (price === '0' || price === 'Infinity' || price === '.' || price === '0.')
+      ? `Price must be higher than 0`
```

Comment:
This is also not optimal, is it? 

Firstt the user gets the message: "higher than 0", and then next he gets the message "higher than x", where x is the current auction clearing price

Reply:
:bulb:  Let me unify the message.